pkg/liqoctl/status/local/pods.go

```diff
@@ -199,6 +200,13 @@ func (pc *PodChecker) Silent() bool {
 // it collects the status of the components of Liqo. The status is
 // collected at the pod level.
 func (pc *PodChecker) Collect(ctx context.Context) {
+	if pc.options.InternalNetworkDisabled {
+		pc.deployments = slice.RemoveString(pc.deployments, liqconsts.LiqoNetworkManagerName)
```

Comment:
It could probably be easier to define them in two separate lists and merge them if the internal network is enabled